src/content/podcast/2024/ep46_nextflow_vs_wdl.md

```diff
@@ -0,0 +1,74 @@
+---
+title: "Nextflow meets WDL: Comparing modern workflow languages"
+episode: 46
+description: Exploring what Nextflow can learn from WDL
+date: 2024-09-17
+type: podcast
+subtype: Technical discussion
+youtubeid: qeJqdV_d8vI
+image: /img/podcast_ep46.png
+tags: Nextflow
+author: Developer advocates
+icon: logo_podcast_channels.jpg
+---
+
+In this episode of Channels, the Nextflow podcast, host Geraldine Van der Auwera from Seqera and her colleague Ben Sherman dive into the differences and similarities between Nextflow and the [Workflow Description Language](https://openwdl.org/) (WDL). They discuss the origins and development of WDL at the Broad Institute, the challenges of working with different workflow languages, and how Nextflow's channel-based data flow model compares to WDL's approach. The conversation covers key features, type systems, and runtime specifications, highlighting the strengths and limitations of both languages. Ben hints at upcoming improvements and enhancements for Nextflow, aimed at making the language more robust and user-friendly. The episode offers valuable insights for bioinformaticians and developers navigating the complexities of workflow management.
+
+<!-- end-archive-description -->
```

Comment:
Side note: Generally don't need these, it'll cap the description after the first paragraph automatically (or a certain character limit). This comment is only needed if you want it to be capped early.

But doesn't hurt to include either..

Reply:
Oh I was wondering why they weren't used consistently in the past ones